server/etcdserver/apply/uber_applier.go

```diff
@@ -130,19 +132,29 @@ func (a *uberApplier) dispatch(r *pb.InternalRaftRequest, shouldApplyV3 membersh
 	switch {
 	case r.Range != nil:
 		op = "Range"
-		ar.Resp, ar.Trace, ar.Err = a.applyV3.Range(r.Range)
+		ctx, trace := traceutil.EnsureTrace(context.Background(), a.lg, op)
```

Comment:
Please move `context.Background()` to outside of switch to encourage other branches to also use it.